CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Compatibility
+
+| CHT Android | Android OS | cht-core |
+| -- | -- | -- |
+| 1.x | 5.0+ | 3.x and 4.x |
```

Comment:
Cool, I think this further clarified a question I had yesterday (that I thought I understood)! So if you use Android 1.0, we will test it against cht-core 3.x. Am I understanding that correctly?

Reply:
Yes. When cht-android 1.0 (and future 1.x releases) are being release tested we'll have to test it against cht-core 3.x and 4.x.